packages/terra-list/src/ListItem.jsx

```diff
@@ -94,6 +100,10 @@ const ListItem = ({
     attrSpread['data-item-show-focus'] = 'true';
     attrSpread.onBlur = ListUtils.wrappedEventCallback(onBlur, event => event.currentTarget.setAttribute('data-item-show-focus', 'true'));
     attrSpread.onMouseDown = ListUtils.wrappedEventCallback(onMouseDown, event => event.currentTarget.setAttribute('data-item-show-focus', 'false'));
+  } else if (isRoleListbox) {
+    attrSpread.role = 'option';
+    attrSpread['aria-selected'] = isSelected;
+    attrSpread['aria-disabled'] = 'true';
```

Comment:
@neilpfeiffer Should we be applying disabled styles for this concept or is the non-selectable styles enough to differentiate this non-clickable item behavior?

Reply:
@neilpfeiffer any updates on this disabled styles ?